From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Patrick McCarty <patrick.mccarty@intel.com>
Date: Tue, 2 Jul 2019 14:25:23 -0700
Subject: [PATCH] Disable tcltk namespace import

The R package in Clear Linux OS installs the tcltk package, but it does
not work, so disable the namespace import to fix the build.

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 sm/NAMESPACE | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sm/NAMESPACE b/sm/NAMESPACE
index 0ea8cdc..cf695a8 100755
--- a/sm/NAMESPACE
+++ b/sm/NAMESPACE
@@ -21,7 +21,7 @@ export(binning, britmap, hcv, hnorm, hsj, nise, nmise, nnbr,
              "qchisq", "quantile", "rbeta", "rbinom", "residuals",
              "rgamma", "rpois", "sd", "terms", "uniroot", "var")
   importFrom("utils", "menu")
-  import(tcltk)
+# import(tcltk)
 # import(tkrplot)
 # import(rpanel)
 #   importFrom("rpanel", "rp.plot3d", "rp.do", "rp.tkrplot", "rp.tkrreplot",
